test/clj_gatling/core_test.clj

```diff
@@ -45,22 +50,22 @@
                                                    {:requests 500
                                                     :concurrency 1})]
     (force-stop-fn)
-    (is (< (:ok @results) 250))
-    (is (< (:ko @results) 250))))
+    (is (< (:ok @results) 260))
+    (is (< (:ko @results) 260))))
 
 (deftest simulation-returns-raw-report-from-file
   (let [summary (run (simulation "test-file-raw")
                      {:reporters [raw-reporter/file-reporter]
-                      :requests 10
+                      :requests 100
                       :concurrency 1})]
-    (is (= 10 (count (mapcat :requests (:raw summary)))))))
+    (is (approximately== (count (mapcat :requests (:raw summary))) 100 :accuracy 5))))
 
 (deftest simulation-returns-raw-report-from-memory
   (let [summary (run (simulation "test-memory-raw")
                      {:reporters [raw-reporter/in-memory-reporter]
-                      :requests 10
+                      :requests 100
                       :concurrency 1})]
-    (is (= 10 (count (mapcat :requests (:raw summary)))))))
+    (is (approximately== (count (mapcat :requests (:raw summary))) 100 :accuracy 5))))
```

Comment:
A number of these tests had to be modified from absolute values to approximate checks because creating more runners means that we may be a few more messages over the target, and the % we might be over gets better as we do more requests (because it's a function of the concurrency, rather than the request count).